### PR TITLE
Improve overview/readme/build_integration/clusterfuzzlite (landing page)

### PR DESCRIPTION
**Pull request by jonathanmetzman**

### CODE REVIEW

File: docs/build_integration.md [hunk 1]
```diff
@@ -73,6 +73,9 @@ $ cd /path/to/oss-fuzz
 $ export PATH_TO_PROJECT=<path_to_your_project>
 $ python infra/helper.py generate --external --language=c++ $PATH_TO_PROJECT
 ```
+Note that you may need to change the `--language` argument to another value
+if your project is written in another language, this is discussed more in the
```
Comment (olivekl) [inline docs/build_integration.md:77]: Just a grammar nit. Either change the comma after "language" to a period, or put the second clause in parentheses like this:

`Note that you may need to change the `--language` argument to another value if your project is written in another language (discussed more in the...)`

File: docs/build_integration.md [hunk 2]
```diff
@@ -93,11 +96,13 @@ Programming language the project is written in. Values you can specify include:
...
 * [`python`]({{ site.baseurl }}//build-integration/python-lang/)
-* [`jvm` (Java, Kotlin, Scala and other JVM-based languages)]({{ site.baseurl }}//build-integration/jvm-lang/)
-* [swift]({{ site.baseurl }}//build-integration/swift-lang/)
+* [`jvm`]({{ site.baseurl }}//build-integration/jvm-lang/)
+  * This should be used for Java, Kotlin, Scala and other JVM-based languages.
+* [`swift`]({{ site.baseurl }}//build-integration/swift-lang/)
 
 Most of this guide applies directly to C/C++ projects. Please see the relevant
 subguides for how to build fuzzers for that language.
+Note that `c` and `c++` are the same to ClusterFuzzLite.
```
Comment (olivekl) [inline docs/build_integration.md:105]: Should C and C++ be capitalized?

**Review by olivekl — COMMENTED** [review]
LGTM with minor comments!

**Comment by jonathanmetzman** [comment]
Nope.

**Pull request merged**